core/src/test/java/io/questdb/test/griffin/CreateTableDedupTest.java

```diff
@@ -162,6 +162,28 @@ public void testCreateTableSetTypeSqlSyntaxErrors() throws Exception {
         });
     }
 
+    @Test
+    public void testCreateTableWithDoubleQuotes() throws Exception {
+        String tableName = "ABB_Events2";
+        assertMemoryLeak(ff, () -> {
+            ddl(
```

Comment:
Ugh! String literal  used as table name :)

Reply:
I meant the single quotes  around table name in sql command. It should be double quotes instead .

Reply:
ah!, nope, we do not make distinction, see isQuote method. I would not want to be pedantic in this specific meaningless "issue" when the codebase makes no distinction between quotes:

```
public static boolean isQuote(char c) {
        switch (c) {
            case '\'':
            case '"':
            case '`':
                return true;
            default:
                return false;
        }
    }
```

it is a case of take it or leave it. :)